src/test/groovy/site/MenuSpec.groovy

```diff
@@ -0,0 +1,56 @@
+package docToolchain
+import spock.lang.*
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+
+class MenuSpec extends Specification {
+
+    class ContentFixture {
+        def menu
+        def entriesMap
+        def newEntries
+    }
+
+    void 'test empty published content'() {
+        given: 'empty published_content'
+            Binding binding = new Binding()
+            binding.published_content = []
+        when: 'run the `menu.groovy` script'
+            runMenuScript(binding)
+        then: 'arrays are empty'
+            binding.content.menu == [:]
+            binding.content.entriesMap == [:]
+            binding.content.newEntries == []
+    }
+
+    void 'test with published content'() {
+        given: '3 pages in published_content'
+            Binding binding = new Binding()
+            binding.published_content = [
+                ['jbake-menu': 'foo', 'jbake-title': 'Lorem Ipsum', 'jbake-order': 10, uri : 'foo/10_lorem-ipsum.html'],
```

Comment:
The `order` seems to be of type `String` in the array in reality

Reply:
yes, I think I already fixed this somewhere...

Reply:
Maybe you think of https://github.com/docToolchain/docToolchain/pull/803

But internally in this array the order is a `String`. I think it is converted to `int` later